manual/guides/writing/report.md

```diff
@@ -37,3 +37,19 @@ guidelines.](http://eprints.rclis.org/7469/2/index.html)
 ## After Acceptance
```

Comment:
Acceptance of what?

Reply:
I'm not really sure, that was already a part of the report guide. I believe it's referring to the report being accepted by Prof. Huff, but I'm not very familiar with that process

Reply:
I made the mistake of basing the branch I used for the software template off of the one I was using for the report template so that bled over into this PR on accident.